scripts/localization.ttslua

```diff
@@ -665,42 +1262,42 @@ LOCALIZED_STRINGS = {
     ["UI_AI_CALCULATING"] = "AI正在想下一步",
 
     --Control Panel Global
-
+
     ["TOOLTIP_ENABLE_ALL"] = "勾選所有選項",
-    ["LABEL_TILES"] = "牌",
-    ["TOOLTIP_NUM_TILES"] = "{n1}卡牌",
-    ["LABEL_PREVIOUS_PAGE"] = "上一頁",
+    ["LABEL_TILES"] = "牌",
+    ["TOOLTIP_NUM_TILES"] = "{n1}卡牌",
+    ["LABEL_PREVIOUS_PAGE"] = "上一頁",
     ["LABEL_NEXT_PAGE"] = "下一頁",
-
+
     --Control Panel pg. 1
-
+
         --tooltips
 
-    ["TOOLTIP_AI_HOTSEAT_TOGGLE"] = "按下按鈕來選擇AI難易度及有無:\nAI容易\nAI難\n熱座\n無",
-    ["TOOLTIP_BASE_STARTING_TILE"] = "原始基本卡牌。",
-    ["TOOLTIP_RIVER"] = "12張河流卡牌，在正常卡牌加入之前。",
-    ["TOOLTIP_RIVER_II"] = "12張河流卡牌上若有分流，記號即把河流分成2等份，牧豬記號則給予農夫額外分數。",
-    ["TOOLTIP_RIVER_BB5"] = "12張河流卡牌標有給牧羊人加分的額外羊隻圖案。",
-    ["TOOLTIP_CITY_OF_CARCASSONNE"] = "卡卡頌市起始大卡牌-當您幫他人得分時，您可以選擇放置米寶在卡卡頌市大卡牌的任何一區。這些米寶可以在地標完成時，計分前移位到該區，得到/贏得分數。",
+    ["TOOLTIP_AI_HOTSEAT_TOGGLE"] = "按下按鈕來選擇AI難易度及有無:\nAI容易\nAI難\n熱座\n無",
+    ["TOOLTIP_BASE_STARTING_TILE"] = "原始基本卡牌。",
+    ["TOOLTIP_RIVER"] = "12張河流卡牌，在正常卡牌加入之前。",
+    ["TOOLTIP_RIVER_II"] = "12張河流卡牌上若有分流，記號即把河流分成2等份，牧豬記號則給予農夫額外分數。",
+    ["TOOLTIP_RIVER_BB5"] = "12張河流卡牌標有給牧羊人加分的額外羊隻圖案。",
+    ["TOOLTIP_CITY_OF_CARCASSONNE"] = "卡卡頌市起始大卡牌-當您幫他人得分時，您可以選擇放置米寶在卡卡頌市大卡牌的任何一區。這些米寶可以在地標完成時，計分前移位到該區，得到/贏得分數。",
     ["TOOLTIP_WHEEL_OF_FORTUNE"] = "抽卡牌並旋轉命運之輪來得到機會。",
-
```

Comment:
Appears to be a bunch of whitespacing changes from this point on. Don't worry about fixing it if it isn't easy to exclude them.

Reply:
> I noticed in the diff that GitHub sees a lot of changes in the Chinese strings as well, but I did not touch those. No idea why it thinks these were changed.

Oh I just noticed you mentioned this already. Don't worry about it.